.travis.yml

```diff
@@ -64,6 +64,7 @@ env:
         - CONF="--disable-abi-compat"
         - CONF="--enable-schedule-sp"
         - CONF="--enable-schedule-iquery"
+        - CONF="--enable-dpdk-zero-copy"
```

Comment:
Thanks for the review! Travis downloads the code directly from the DPDK repo and '--with-dpdk-path' is always set while testing these config options.

One thing I noticed while testing this is that DPDK and Netmap are rebuilt for every test, which takes quite a long time. Avoiding this could shave off several minutes from each individual test.